setup.py

```diff
@@ -356,7 +356,7 @@ def run(self):
     },
     extras_require={
         # On Cloud TPU VM install with:
-        # pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/libtpu_releases.html
+        # pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
why did this changed?

Reply:
This is a minor implementation detail. `libtpu_releases.html` and `index.html` are almost identical, except `index.html` is not populated with libtpu versions on the days when the build is failing. `libtpu_releases.html` has a line for every day that points to the last passing build, which may be days old if libtpu is broken.

Reply:
should we update this to the master as well?

Reply:
Yeah, I'll backport the setup.py changes to master